.github/workflows/chatops_retest.yaml

```diff
@@ -0,0 +1,81 @@
+# The _chatops_retest workflow reruns failed GHA for a PR
```

Comment:
Rather than each project duplicating the slash command and retest workflows, would it make sense to manage them centrally and reuse them?

This is the approach I've taken for the Dashboard for example, consuming the reusable workflow from the plumbing repo: https://github.com/tektoncd/dashboard/blob/e0eba748b3dddcdf5493d2353d07d79f3bc9a6dd/.github/workflows/chatops-retest.yml

Reply:
@AlanGreene ah definitely ! 👍🏼